### ISSUE TITLE
Sticky Interaction

### ISSUE THREAD

**Issue by BarclayTech** [zenyr/PocoHud3#20]
If I put down an equipment bag or get helped up (anything that has an interaction circle but can't "stick" it), it flashes as if I was using the sticky interaction. Of course right clicking does nothing and doesn't stop it going away.

Only happens once after putting down a bag or getting helped up etc.

**Comment by zenyr** [comment]
Yeah it is, I should've clean up sticky status but it took a bit more hijacking so I ... left like that partly on purpose. Anyway it is a bug, yes.
